src/landpatterns/QFP.stanza

```diff
@@ -3,55 +3,190 @@ defpackage jsl/landpatterns/QFP:
   import core
   import jitx
 
+  import jsl/ensure
   import jsl/design/settings
   import jsl/landpatterns/leads
   import jsl/landpatterns/quad
+  import jsl/landpatterns/packages
+  import jsl/landpatterns/pad-planner
+  import jsl/landpatterns/numbering
 
+doc: \<DOC>
+QFN Lead Type
+<DOC>
+public defstruct QFP-Lead <: SMT-Lead:
+  lead-type:LeadProtrusion with: (
+      as-method => true,
+      default => SmallGullWingLeads(), ; TODO: is this OK?
```

Comment:
Fine for now - see the `SOIC.stanza` file for how I callout specifics on what lead profile to use.